docs/leios-design/README.md

```diff
@@ -707,32 +708,6 @@ The following non-functional requirements ensure the implementation meets perfor
 
 Note that the PoP checks probably are done at the certificate level, and that the above-described API should not be responsible for this. The current code on BLS12-381 already abstracts over both curves `G1`/`G2`, we should maintain this. The `BLST` package also exposes fast verification over many messages and signatures + public keys by doing a combined pairing check. This might be helpful, though it's currently unclear if we can use this speedup. It might be the case, since we have linear Leios, that this is never needed.
 
-## Performance & Tracing (P&T)
```

Comment:
The content here is still very good and we should consider keeping it! Maybe we will (not need to be in this PR) expand it into _doing_ what is mentioned there? e.g. drafting that specification of relevant traces and their semantics in this section of the "Technical design" chapter.